reviews/views.py

```diff
@@ -0,0 +1,78 @@
+import json, re, bcrypt, jwt
+
+from django.views           import View
+from django.http            import JsonResponse
+from django.db.models       import Avg, Q
+
+from reviews.models         import Review, ReviewImage
+from users.models           import User, WishList
+from users.utils            import login_decorator
+from restaurants.models     import Restaurant
+
+class ReviewCreatView(View):
```

Comment:
view 를 분리한 이유가 있나요?

Reply:
URI를 다르게 쓰려고 분리했습니다

Reply:
리뷰를 생성시에는 restaurant/<int:restaurant_id>/review 를 사용,
리뷰를 수정 및 삭제시에는 restaurant/<int:restaurant_id>/review/<int:review_id>를 사용하고자 하여 view를 분리했습니다.

Reply:
하나의 View class 안에 `get()`, `post()`, `delete()`, `put()` handler method를 구현해주세요. Class형 View의 컨셉에 맞게 ReviewView 객체가 생성, 조회, 삭제, 업데이트 행동을 할 수 있도록 구현해주세요.

```suggestion
class ReviewView(View):
    def get(self, ..): ..
    
    def post(self, ..):
    
    def put(self, ..):
    
    def delete(self, ..):
    
```

Reply:
넵